src/platform/forms-system/src/js/components/ReviewBoxField.jsx

```diff
@@ -0,0 +1,474 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Sentry from '@sentry/browser';
+
+import {
+  getDefaultFormState,
+  getDefaultRegistry,
+} from '@department-of-veterans-affairs/react-jsonschema-form/lib/utils';
+
+import recordEvent from 'platform/monitoring/record-event';
+import { errorSchemaIsValid } from 'platform/forms-system/src/js/validation';
+
+import set from 'platform/utilities/data/set';
+import get from 'platform/utilities/data/get';
+import omit from 'platform/utilities/data/omit';
+import { isReactComponent } from 'platform/utilities/ui';
+
+/**
+ * Displays a review card if the information inside is valid.
+ *
+ * For use on a schema of type 'object' or 'array'.
+ * Intended to wrap objects or arrays to avoid duplicate functionality here.
+ */
+export default class ReviewBoxField extends React.Component {
```

Comment:
I believe @nathanielbuck had something similar on his branch that I looked at earlier. I'm going to hold off on reviewing this since I suspect it may be a mistake.

Reply:
Yeah, @FernandoBoza had copied some in-progress code of mine to use for testing purposes since we're working on similar stuff in parallel. The version in my PR is a little more updated, so feel free to leave any comments or suggestions for improvement there: https://github.com/department-of-veterans-affairs/vets-website/pull/18099